pages/google_search.py

```diff
@@ -13,12 +13,13 @@ class GoogleSearch(Page):
     def __init__(self, driver):
         Page.__init__(self, driver)
         self.driver = driver
-        
+
         self._search_field_locator = (By.ID, 'lst-ib')
```

Comment:
why is this stored as property? i think it should be better pass arguments directly in particular  methods

```
@property
def search_button(self):
      return self.driver.find_element(By.NAME, 'btnG')
```